<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="8910" started-at="2021-06-20T15:35:28 MYT" finished-at="2021-06-20T15:35:37 MYT">
    <groups>
    </groups>
    <test name="Test" duration-ms="8910" started-at="2021-06-20T15:35:28 MYT" finished-at="2021-06-20T15:35:37 MYT">
      <class name="shoppingTest.LoginTestSD">
        <test-method status="PASS" signature="setup()[pri:0, instance:shoppingTest.LoginTestSD@fad74ee]" name="setup" is-config="true" duration-ms="2510" started-at="2021-06-20T15:35:28 MYT" finished-at="2021-06-20T15:35:30 MYT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LoginTest(java.lang.String,java.lang.String)[pri:0, instance:shoppingTest.LoginTestSD@fad74ee]" name="LoginTest" duration-ms="603" started-at="2021-06-20T15:35:30 MYT" finished-at="2021-06-20T15:35:31 MYT">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="AddToCart()[pri:0, instance:shoppingTest.LoginTestSD@fad74ee]" name="AddToCart" duration-ms="219" started-at="2021-06-20T15:35:31 MYT" depends-on-methods="shoppingTest.LoginTestSD.LoginTest" finished-at="2021-06-20T15:35:31 MYT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddToCart -->
        <test-method status="FAIL" signature="teardown()[pri:0, instance:shoppingTest.LoginTestSD@fad74ee]" name="teardown" is-config="true" duration-ms="5508" started-at="2021-06-20T15:35:31 MYT" finished-at="2021-06-20T15:35:37 MYT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [PRODUCTS--wrong] but found [PRODUCTS]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [PRODUCTS--wrong] but found [PRODUCTS]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at shoppingTest.LoginTestSD.teardown(LoginTestSD.java:206)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.afterRun(TestRunner.java:877)
at org.testng.TestRunner.run(TestRunner.java:599)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- shoppingTest.LoginTestSD -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
